troposphere/static/js/modals/instance/resume.js

```diff
@@ -1,16 +1,31 @@
 import ModalHelpers from "components/modals/ModalHelpers";
-
 import InstanceResumeModal from "components/modals/instance/InstanceResumeModal";
+
 import actions from "actions";
 
 
 export default {
+    /**
+     * Resume any valid/actionable resources
+     *
+     * The action method handles resources verifying type (via instanceof)
+     * to ensure that multi0ple resources can be handled.
+     *
+     * @param resources - collection of resources to act on
+     */
+    resume: function(resources) {
```

Comment:
If I followed it correctly this used to be a single instance. I think it's dissonant for a reference to be called props.instance and then actually be a list of instances (resources) later. I think we should call it resources earlier in [troposphere/static/js/components/projects/resources/instance/details/actions/InstanceActionsAndLinks.jsx](https://github.com/cyverse/troposphere/blob/3666685472e8598356b77c278f0eb8ba5212784e/troposphere/static/js/components/projects/resources/instance/details/actions/InstanceActionsAndLinks.jsx#L32). See below:
```
    onResume: function() {
        modals.InstanceModals.resume(this.props.instance);
    },
```